templates/institutional-landing/institutional-landing-bad.html

```diff
@@ -6,48 +6,4 @@
 dateModified: "2024-04-16"
 ---
 
-{%- include /design-patterns/gc-intro/gc-intro-image.html -%}
-
-<div class="container mrgn-bttm-lg">
-	{%- include /components/gc-servinfo/3col-hidden-title.html -%}
-</div>
-
-<div class="well well-sm brdr-0 mrgn-bttm-md">
-	<div class="container">
-		<section>
-			<h2>Contact us</h2>
-			<ul class="wb-eqht list-unstyled mrgn-tp-lg lst-spcd-2 colcount-md-3 bold-content">
-				<li><a href="#">Contact [Institution]</a></li>
-				<li><a href="#">[Top contact task 2]</a></li>
-				<li><a href="#">[Top contact task 3]</a></li>
-			</ul>
-		</section>
-	</div>
-</div>
-
-<div class="container">
-	<div class="row">
-		<div class="col-md-8">{%- include /design-patterns/gc-institution/gc-institution.html -%}</div>
-		<div class="col-md-4">{%- include gc-follow-us/gc-follow-us.html -%}</div>
-	</div>
-
-	{%- include /components/gc-minister/gc-minister.html -%}
-
-	<div class="row">
-		<div class="col-md-4">
-			<section class="wb-feeds limit-3">
-				<h2>News</h2>
-				<ul class="feeds-cont list-unstyled lst-spcd">
-					<li>
-						<a data-ajax="https://wet-boew.github.io/GCWeb/components/feeds/demo/example-en.atom.json" href="#">[Institution] news</a>
-					</li>
-				</ul>
-				<p>More: <a href="#">[Institution] news</a></p>
-			</section>
-		</div>
-
-		<div class="col-md-8">
-			{%- include /components/gc-features/two-tiles.html -%}
-		</div>
-	</div>
-</div>
+<div data-ajax-replace="institutional-landing-en.html main > *:not(.pagedetails)"></div>
```

Comment:
This dont' work. The custom JS are not triggered to fix the heading by adding the page body CSS class `page-type-ilp`